programs/create-genesis/ee_genesis/ee_genesis_builder.hpp

```diff
@@ -1,11 +1,13 @@
 #pragma once
 
-#include "golos_dump_container.hpp"
-#include "event_engine_genesis.hpp"
+#include "../golos_state/golos_state_reader.hpp" // Include 1st for custom_unpack
+#include "../common/genesis_info.hpp"
+#include "golos_dump_header.hpp"
```

Comment:
this file contains not only file header, but also it's internal structure info, that's why it was named `golos_dump_container.hpp`. it's not about STL containers, it's file container.

If you don't like "container", please list some other variants. I don't like "header", because it's less precise.